garage/envs/wrappers/fire_reset.py

```diff
@@ -0,0 +1,28 @@
+"""Fire reset wrapper for gym.Env."""
+import gym
+
+
+class FireReset(gym.Wrapper):
+    """
+    Fire reset wrapper for gym.Env.
+
+    Take action "fire" on reset.
+
+    Args:
+        env: The environment to be wrapped.
+    """
+
+    def __init__(self, env):
+        super().__init__(env)
+        assert env.unwrapped.get_action_meanings()[1] == 'FIRE', (
+            "Only use fire reset wrapper for suitable environment!")
+        assert len(env.unwrapped.get_action_meanings()) >= 3, (
+            "Only use fire reset wrapper for suitable environment!")
+
+    def reset(self, **kwargs):
+        """gym.Env reset function."""
+        self.env.reset(**kwargs)
+        obs, _, done, _ = self.env.step(1)
```

Comment:
What do we need FIRE here?

Reply:
In most of the Atari games, the agent has to FIRE in order to start the game (like when you press START button when you play games). This wrapper avoids the agent doing random actions that have no effect on the environment right after environment reset.